test/E2E_Test/MvcControllerTest.cs

```diff
@@ -49,7 +49,7 @@ public void TestControllerGenerators(string baselineFile, string generatedFilePa
             }
         }
 
-        [Fact]
+        [Fact(Skip="Disable flakey tests https://github.com/aspnet/Scaffolding/issues/491")]
```

Comment:
Nit: We typically spell the word as `flaky` rather than `flakey`, though both variants are in the dictionary.